pages/sign/in.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import * as SS from '@/components/_styled/signStyled';
+import SignInForm from '@/components/sign/SignInForm';
+
+export default function SignIn() {
+    const router = useRouter();
+    const [isMounted, setIsMounted] = useState(false);
+
+    useEffect(() => {
+        setIsMounted(true); 
+    }, []);
+
+    useEffect(() => {
+        if (isMounted) {
+            const accessToken = localStorage.getItem('accessToken');
+            if (accessToken) {
+                router.push('/');
+            }
+        }
+    }, [isMounted]);
```

Comment:
P2:
해당 state 대신 isReady 속성을 쓰시면 좋을 것 같습니다.

```suggestion
  useEffect(() => {
    if (router.isReady) {
      const accessToken = localStorage.getItem("accessToken");
      if (accessToken) {
        router.push("/");
      }
    }
  }, [router]);
```